src/Sulu/Bundle/RouteBundle/Resources/config/routing.xml

```diff
@@ -10,7 +10,7 @@
     <services>
         <!-- route-defaults -->
         <service id="sulu_route.routing.defaults_provider" class="Sulu\Bundle\RouteBundle\Routing\Defaults\RouteDefaultsProvider">
-            <argument type="collection">%sulu_route.routing.defaults_provider%</argument>
```

Comment:
This parameter is defined nowhere in the code. So not replacing this would crash the application.